src/Pages/Login/SignIn.js

```diff
@@ -0,0 +1,313 @@
+import React, { useState } from "react";
+import { withRouter } from "react-router-dom";
+import { useHistory } from "react-router";
+import styled from "styled-components";
+import KaKaoLogin from "react-kakao-login";
+
+function SignIn() {
+  const [user, setUser] = useState({
+    identifier: "",
+    password: "",
+  });
+
+  const [kakao, setKakao] = useState({ data: {} });
```

Comment:
```suggestion
  const [kakao, setKakao] = useState({});
```
초기값이 nesting 된 객체여야 하나요?

Reply:
카카오 로그인 부분 전반적인 수정이 필요한거같아서 코드 전체 삭제했습니다 ㅜㅜ